client.go

```diff
@@ -42,26 +43,36 @@ type Client struct {
 	VPCPeeringConnections       *VPCPeeringConnectionsHandler
 }
 
+// SetUserAgent sets default userAgent in case it has not been provided
+func SetUserAgent(userAgent string) string {
```

Comment:
Function name is confusing since this never sets anything, just returns a value. `GetUserAgentOrDefault` or something like that might be more descriptive.